### DISCUSSION TITLE
Release 1.2

### DISCUSSION THREAD

**Discussion by Macronair** [Maerciez/TheMillionaireGame#81] [General]
Hi all,

Version 1.2 is out to use and try it out yourself. Of course it is possible that some bugs or errors are still present (if that's the case, then contact me via this thread), but still I hope the experience of using the software with the new features will be much better than the previous versions.

If there are any questions, suggestions or other reactions to this version, you can drop it here.
And also for the content creators or live streamers, when you use this software for your content, then let me know! Not for permission (you can use it without asking me), but I'm curious to what you guys making of it. 😄 

Have fun and enjoy using version 1.2!

**Comment by JCEurovisionFan** [comment]
I tried to install SQL Server 2022, but I encountered some problems like these:
![image](https://github.com/user-attachments/assets/b7e4c3a6-3285-4161-a5ef-4f7627212033)
![image](https://github.com/user-attachments/assets/4b747767-9700-4647-bbe6-f2b3b1103ed7)
How can I make it work? I can't do SQL databases, I'm afraid.

**Reply by Macronair** [reply]
Can you try downloading SQL Server Express 2022 again from the Microsoft website?
It seems that some packages are missing in the installation files.

I used this link from Microsoft:
https://go.microsoft.com/fwlink/p/?linkid=2216019&clcid=0x413&culture=nl-nl&country=nl

**Comment by MrVideoFreak** [comment]
Do i have to rewrite all my old questions from 1.1.7 to get them into 1.2 as I don't know how to import all my questions in

**Reply by Macronair** [reply]
In most cases it is not necessary. Older databases are still supported in 1.2, but I recommend migrating from SQL LocalDB to SQL Express and - of course - back-up your database before you make major changes.

When you use version 1.2, you have also the option to use the LocalDB databases which were used in the older versions.
Then, in the Questions Editor, you can export them as a CSV file.
After you've done that, log in to the new database (local or remote SQL Express install) and import them with the same questions editor.

Good luck! ;)

**Reply by MrVideoFreak** [reply]
It wouldn't let me import the questions

**Comment by Baumhausbanane123e** [comment]
Just a little well not bug, rather mistake:
In the Host Messages Window, "Phone a friend" is labeld as "Plus One".

**Reply by Macronair** [reply]
Ah, that's one small mistake from my side.
I will patch this soon. Thanks for notifying!
